src/common/dsp/SurgeVoice.cpp

```diff
@@ -215,8 +216,10 @@ SurgeVoice::SurgeVoice(SurgeStorage *storage, SurgeSceneStorage *oscene, pdata *
     const bool isCh23Mode = Surge::Storage::getUserDefaultValue(
         storage, Surge::Storage::UseCh2Ch3ToPlayScenesIndividually, true, false);
     const bool isChSplitMode = storage->getPatch().scenemode.val.i == sm_chsplit;
+    const bool ChannelPerOctave = storage->mapChannelToOctave;
 
-    state.mtsUseChannelWhenRetuning = (mpeEnabled || isChSplitMode || isCh23Mode);
+    state.mtsUseChannelWhenRetuning =
+        (mpeEnabled || isChSplitMode || isCh23Mode || ChannelPerOctave);
```

Comment:
You could just directly use storage->mapChannelToOctave here, no need to have it as a variable when it's only used once.

Reply:
Right, I thought I'd match what the other parts of the check were doing. But that makes sense. Will change.